<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>507</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>631</width>
     <height>431</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="VideoLayout"/>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>450</y>
     <width>631</width>
     <height>21</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="ControlToolLayout"/>
  </widget>
  <widget class="QPushButton" name="startorPauseButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>480</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid #999999;\n;
height:12px; /* the groove expands to the size of the slider by default. by giving it a height, it has a fixed size */
 background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #B1B1B1, stop:1 #c4c4c4);
margin: 2px 0;
border-radius:5px;
\n;
		</string>
   </property>
   <property name="text">
    <string>play</string>
   </property>
  </widget>
  <widget class="QPushButton" name="preButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>480</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid #999999;\n;
height:12px; /* the groove expands to the size of the slider by default. by giving it a height, it has a fixed size */
 background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #B1B1B1, stop:1 #c4c4c4);
margin: 2px 0;
border-radius:5px;
\n;
		</string>
   </property>
   <property name="text">
    <string>Pre</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
